### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by LisaQ04**
Added product detail and order confirmed pages

### CODE REVIEW

File: src/Components/Shop/OrderConfirmedComponent.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    static navigationOptions(){
+        return {
+            title: 'Order Confirmed'
+        };
+    } 
+    
+    render() {
+        return ( 
+            <View style={styles.container}> 
+                <View style={{ width: 415, height: 300,backgroundColor:'#006400' }}>
```
Comment (GeromeSchutte) [inline src/Components/Shop/OrderConfirmedComponent.js]: please separate the styles

File: src/Components/Shop/ProductDetailComponent.js [hunk 2]
```diff
@@ -1,29 +1,100 @@
...
-
-        </View>;
+        return ( 
+            <View style={styles.container}> 
+                <View style={{ paddingTop:15 }}>
+                    <Image source={HomeCover}
+                        style={{ width: 300, height: 200 }}>
+                    </Image> 
+                </View>
+                <View style={{ flexDirection:'row', paddingTop:25 }}>
```
Comment (GeromeSchutte) [inline src/Components/Shop/ProductDetailComponent.js]: please separate the styles

File: src/Components/Shop/OrderConfirmedComponent.js (hunk 1 above)
Comment (LisaQ04) [reply src/Components/Shop/OrderConfirmedComponent.js]: Separated the styles.

File: src/Components/Shop/ProductDetailComponent.js (hunk 2 above)
Comment (LisaQ04) [reply src/Components/Shop/ProductDetailComponent.js]: Separated the styles.

**Pull request merged**